ts/src/bitmex.ts

```diff
@@ -1166,14 +1166,15 @@ export default class bitmex extends Exchange {
             // for unrealized pnl and other transactions without a timestamp
             timestamp = 0; // see comments above
         }
+        let fee = undefined;
         let feeCost = this.safeString (item, 'fee');
         if (feeCost !== undefined) {
             feeCost = this.convertToRealAmount (code, feeCost);
+            fee = {
+                'cost': this.parseToNumeric (feeCost),
```

Comment:
@sc0Vu I don't think we should be using `parseToNumeric` here, we should probably use parseNumber

Reply:
`parseToNumeric` should be used to build up the request